src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/Nodes/NodePropertyFilter.cs

```diff
@@ -276,6 +293,9 @@ public enum FilterType
         LESSTHANOREQUAL,
         GREATERTHANOREQUAL,
         BETWEEN,
-        NOTBETWEEN
+        NOTBETWEEN,
```

Comment:
Shouldn't there just be between? The others don't have their NOT versions here

Reply:
That's because not between works in a different way than other not versions.